modules/ossm-rn-new-features.adoc

```diff
@@ -131,6 +153,31 @@ spec:
 ----
 Restricting route attachment on Gateway API listeners is possible using the `SameNamespace` or `All` settings. Istio ignores usage of label selectors in `listeners.allowedRoutes.namespaces` and reverts to the default behavior (`SameNamespace`).
 
+
+
+
+== New features {SMProductName} 2.1.5
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), and is supported on OpenShift Container Platform 4.9 or later.
+
+=== Component versions included in {SMProductName} version 2.1.6
```

Comment:
Verify version.  Is is 2.1.6 or 2.1.5.1?

Reply:
@jshaughn the PR has been updated based on the information discussed in the program call. The revised preview build is also available. Thanks for bringing this up so that it could be clarified.